interface/usergroup/usergroup_admin.php

```diff
@@ -103,6 +103,19 @@
               $tqvar = formData('mname','P');
               sqlStatement("update users set mname='$tqvar' where id= ? ", array($_POST["id"]));
       }
+      if ($_POST["role_name"]) {
+        $tqvar = formData('role_name', 'P');
+        sqlStatement("update users set fullscreen_role='$tqvar' where id= ?", array($_POST["id"]));
```

Comment:
Good this is sql binding

Reply:
One is bound, $tqvar is not, right?  (these are questions about binding)